src/Scripts/Models/ChatHistoryItem.gd

```diff
@@ -31,6 +31,7 @@ var Role: ChatRole:
 var InjectedNote: String:
 	set(value): SingletonObject.save_state(false); InjectedNote = value
 
+@warning_ignore("shadowed_global_identifier")
```

Comment:
Rename this varible

Reply:
Fixed by f790b477e4b8420edfa01fda77d4f0dc94ca7c9d